subsys/net/lib/openthread/platform/shell.c

```diff
@@ -25,8 +25,19 @@ static bool is_shell_initialized;
 
```

Comment:
LGTM. Minor nit about a typo in the commit msg `s/promt prining/prompt printing/`

Reply:
Thanks, fixed

Reply:
There is still the `promt` there, please fix that too

Reply:
Indeed, sorry for the fuss, fixed now